lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php

```diff
@@ -75,6 +92,26 @@ protected function initialize(InputInterface $input, OutputInterface $output) :
             $this->dependencyFactory->setConfigurationLoader($configurationLoader);
         }
 
+        if ($input->getOption('em')!== null && $input->getOption('conn')!==null) {
+            throw new InvalidOptionUsage('You can specify only one of the --em and --conn options.');
+        }
+
+        $connectionLoader = $this->dependencyFactory->getConnectionLoader();
+        if ($connectionLoader instanceof ConnectionRegistryConnection && $input->getOption('conn')!==null) {
+            $connectionName = $input->getOption('conn');
+            $connectionLoader->setConnectionName((string) $connectionName);
+        } elseif ($input->getOption('conn')!==null) {
+            throw new InvalidOptionUsage('You can use the --conn option only if the connection loader is of type ConnectionRegistryConnection.');
+        }
+
+        $connectionLoader = $this->dependencyFactory->getEntityManagerLoader();
+        if ($connectionLoader instanceof ManagerRegistryEntityManager && $input->getOption('em')!==null) {
+            $emName = $input->getOption('em');
+            $connectionLoader->setManagerName((string) $emName);
```

Comment:
At this point `$input->getOption('em')` can't return a null value.

Reply:
That is true, but phpstan detects that can be `int|string`, so a casting to string is needed (i guess that happens if you name your entity manager something as `792`, passing `--em=792`, it seems that `792` will be an integer.... (i did not test it  but phpstan says so...)